### DISCUSSION TITLE
Math courses

### DISCUSSION THREAD

**Discussion by RazaGR** [mrdbourke/pytorch-deep-learning#628] [General]
Hey @mrdbourke , Thanks a lot for putting all this together, never found such an informative course on ML/DL.
I wanted to ask if you can suggest a course  for learning math in ML? as easy as your videos. I would like to read research papers but math explained in those papers has been my biggest problem.

Thanks

**Comment by mrdbourke** [comment]
Hey @RazaGR , 

I don't know of any math courses with the same style as my videos, however, the videos by 3blue1brown are excellent.

See the deep learning playlist here: https://www.youtube.com/watch?v=aircAruvnKk 

There's also the Math for Machine Learning book which is a great reference: https://mml-book.github.io/

**Comment by shisirkha** [comment]
Yeah 3blue1brown videos are awesome but sometimes pretty complicated.
